packages/core/src/renderers/systems/filter/FilterSystem.js

```diff
@@ -110,9 +118,7 @@ export default class FilterSystem extends WebGLSystem
 
         // round to whole number based on resolution
         // TODO move that to the shader too?
-        state.sourceFrame = target.filterArea ? this.transformFilterArea(this.tempRect,
-            target.filterArea,
-            target.transform) : target.getBounds(true);
+        state.sourceFrame = target.filterArea || target.getBounds(true);
 
```

Comment:
Been thinking about this one a lot.. I think we should keep the transform on the filter area. My thinking comes that the purpose of filter area is to optimise away the getBounds function. Pretty much the same way we do it with hit area. This brings the two behaviours in line and I believe It will be more intuitive to pixi users for it. What do peeps think?

Say for example I add a filterArea to an object, as the object moves around the filter area would follow it.

Reply:
I dont want to change existing behaviour of `element.filterArea = app.screen`, I posted that line too much, and now it'll be broken.